umu/umu_run.py

```diff
@@ -239,6 +251,9 @@ def build_command(
         err: str = "The following file was not found in PROTONPATH: proton"
         raise FileNotFoundError(err)
 
+    if FLATPAK_PATH and Path(__file__).resolve().parent == Path("/app/share"):
```

Comment:
Why both clauses? what does the second one achieve? Is it required? Seems like it is hardcoding a path which can be bad if it isn't explicitly required or can work without.

Reply:
There's actually a typo there -- it's actually /app/share/umu. 

I want to check for the entry point to ensure that the Flatpak application is being launched instead of just trusting the path/environment variable inputs.

Reply:
I still find it superfluous, if the user wants to mess with variables or paths, this won't stop them, but it's not important enough to  argue over it. I am fine with it.